spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -0,0 +1,113 @@
+[[api-guide]]
+= REST API Guide
+
+[partintro]
+--
+In this section you will learn all about Spring Cloud Data Flows REST API.
```

Comment:
Moreover, and this goes beyond this simple PR, we should settle on Data Flow (here) or Dataflow (line 15), but not both